taipy/core/scenario/_scenario_manager.py

```diff
@@ -261,6 +261,17 @@ def _get_all_by_cycle(cls, cycle: Cycle) -> List[Scenario]:
             fil.update({"cycle": cycle.id})
         return cls._get_all_by(filters)
 
+    @classmethod
+    def _get_scenarios_by_time(cls, Year: int, Month: int, Day: int, Hour: int, Minute: int) -> List[Scenario]:
+        duration = datetime.datetime(Year, Month, Day, Hour, Minute)
+
+        scenarios = []
+        for scenario in cls._get_all():
+            cycle = scenario.cycle
+            if cycle.start_date <= duration and duration <= cycle.end_date:
```

Comment:
I believe the logic is quite vague here.

What if the user wants to get scenarios that are executed on the 15th every month or something similar to that?

Reply:
Okay , I am still trying to build a logic for this specific use case , but for now I was hoping the dates could be precomputed and passed to  to the `_get_scenarios _by_time`  in a for loop.
```py

all_scenarios_on_the_15 = []
weekdays[1,4,5,1,3,6,1,4,7,2,5,7]
for i in range (1,12):
    scenario_on_the_15 = tp.get_scenarios_by_time(2024,15, weekdays[i],0,0)
    all_scenarios_on_the_15extend(scenario_on_the_15) 
```

I also thougth We could leave the task of precomputed dates to the UI. For example a  a week or month or date picker .  
  <input type="week" id="week" name="week">
![image](https://github.com/Avaiga/taipy/assets/24577149/f4436c85-5e26-41f5-abce-233fc56bcdd6) 
The idea is , the dates could be parsed from the UI and passed to the `get_scenarios_by_time()` function.  I could modify the function to take an array of dates (strings) . Is this intuition okay ?